src/components/InfoDialog.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div
+        class="q-pa-md q-gutter-sm"
+    >
+        <q-dialog
+            v-model="showDialog"
+        >
+            <q-card
+                class="window-width"
+                id="help-dialog"
+            >
+                <q-card-section class="row items-center q-pb-none">
+                    <div class="text-h5">
+                        {{ data.title }} field
+                    </div>
+                    <q-space />
+                    <q-btn
+                        icon="close"
+                        flat
+                        round
+                        dense
+                        v-close-popup
+                    />
+                </q-card-section>
+
+                <q-card-section>
+                    {{ data.description }}
+                </q-card-section>
+                <q-card-section>
```

Comment:
```suggestion
                <q-card-section v-if="data.examples">
```
I would make this optional since sometimes there are no examples (radio for types, for instance).